docs/release/trg-4/trg-4-02.md

```diff
@@ -23,6 +20,9 @@ As Eclipse Tractus-X project, we don't have automated processes for publishing c
 DockerHub is running container scans for all [official images](https://docs.docker.com/trusted-content/official-images/)
 and is publishing the scans result in the [docker-library/repo-info repository](https://github.com/docker-library/repo-info).
 
+Utilizing images from DockerHub is advantageous due to its role as a trusted repository of curated and verified open source containers.
```

Comment:
I think this is a bit misleading. This sounds like pulling images from DockerHub is always safe, because every single image is meeting security best practices. This is absolutely not the case.
This is only true (and also not to full extend) for [official DockerHub images](https://docs.docker.com/trusted-content/official-images/)
Even when using official images, you are not safe from vulnerabilities. This is highly dependent on the time you pull an image

Reply:
Right, made it more precise: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/527/commits/a53681394867b418386b9b0d28ecd0d14ee763e7

> Even when using official images, you are not safe from vulnerabilities. This is highly dependent on the time you pull an image

Of course however that's not only applicable image wise but in general, software wise context.